textRevers/src/main/java/app/textRevers/CreateOutputLook.java

```diff
@@ -0,0 +1,25 @@
+package app.textRevers;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class CreateOutputLook {
```

Comment:
There is no need to create new class FOR ONE METHOD. It can be put in the main or previous class ass the second public method.